app/javascript/logs/Logs.vue

```diff
@@ -36,25 +36,25 @@ removeQueryParams(); // remove query params such as `new_entry` and `auth_token`
 renderBootstrappedToasts();
 
 onMounted(() => {
-  if (!isSharedLog.value) {
+  if (!isSharedLogView.value) {
     // If we are viewing a specific log, we want to ensure that the log entries for that log are
     // fetched first, so delay 10ms.
     // Otherwise (i.e. if viewing index), fetch all entries immediately.
     const delayBeforeFetchingAllLogs = selectedLog.value ? 10 : 0;
     setTimeout(() => {
       logsStore.fetchAllLogEntries();
     }, delayBeforeFetchingAllLogs);
-  }
 
-  document.addEventListener('keydown', (event) => {
-    if (
-      event.key === 'k' &&
-      (event.metaKey === true || event.ctrlKey === true) // Meta for macOS, Ctrl for Windows/Linux
-    ) {
-      event.preventDefault(); // Prevent default behavior for the shortcut
-      modalStore.showModal({ modalName: 'log-selector' });
-    }
-  });
+    document.addEventListener('keydown', (event) => {
```

Comment:
(Don't register the keyboard shortcut listener when viewing the log via a share view.)